src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -222,6 +222,72 @@ export default defineConfig({
 });
 ```
 
+### isr
+
+**Type:** `boolean | VercelISRConfig`<br/>
+**Available for:** Serverless
+<Since v="7.2.0" pkg="@astrojs/vercel" />
+**Default**: 'false'
+
+Allows your project to be deployed as an [ISR (Incremental Static Regeneration)](https://vercel.com/docs/incremental-static-regeneration) function, which caches your on-demand rendered pages in the same way as prerendered pages after first request. Note that ISR function requests do not include search params, similar to [requests](/en/reference/api-reference/#astrorequest) in static mode.
+
+```js title="astro.config.mjs" ins={8}
+import { defineConfig } from 'astro/config';
+import vercel from '@astrojs/vercel/serverless';
+
+export default defineConfig({
+  // ...
+  output: 'server',
+  adapter: vercel({
+    isr: true,
+  }),
+});
+```
+
+### VercelISRConfig
+
+By default, an ISR function caches for the duration of your deployment. You can further control caching using `bypassToken`, `exclude`, and `expiration`.
+
+```js title="astro.config.mjs" ins={8}
```

Comment:
I think this `ins` is left over from a copy-paste of the previous code block? I think it makes less sense in this example.

````suggestion
```js title="astro.config.mjs"
````